keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -646,7 +646,7 @@ kpxc.rememberCredentialsFromContextMenu = async function() {
 // Credential Banner can force the retrieval for reloading new/modified credentials.
 kpxc.retrieveCredentials = async function(force = false) {
     if (!await isIframeAllowed()) {
-        return [];
+        return;
```

Comment:
Not sure why this is changed. We prefer returning an empty list if there are no credentials.

Reply:
This function doesn't return anything

Reply:
Hah, true.